### PR TITLE
[Doc Improvement][ICM update for gov environment for shared channel apps][3966093]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/edfcb231ee3d34fc18f2955061bdec3934a259c2/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b35f8648-1e55-4f7f-853e-1e35032abb2c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3984760dc42243687d41147237b9fea28fce936e/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d3215835-b421-49d9-8f6c-9629fa465145) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/shared-channels.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ Microsoft Teams Connect shared channels allow members of a channel to collaborat
 * Individuals and other teams of other organizations.
 
 > [!NOTE]
-> Tab apps in shared channels are available in [Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
+> * Tab apps in shared channels are available in [Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
+> * SharePoint pages and SharePoint apps in shared channels aren't supported in GCC, GCC-High, and DOD environments.
```
Comment (jacobdevera) [inline msteams-platform/concepts/build-and-test/shared-channels.md:21]:
```suggestion
> * The SharePoint and SharePoint Pages apps aren't currently supported for shared channels in Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD) environments.
```

Comment (v-npaladugu) [reply msteams-platform/concepts/build-and-test/shared-channels.md:21]: - Branding is SharePoint pages (https://learn.microsoft.com/en-us/sharepoint/dev/solution-guidance/sharepoint-pages-and-the-page-model)

- We don't use the term 'currently' in our doc set

- Abbreviations or acronyms are only expanded in the fist instance of the article. Use the abbreviation/ acronym only for the ret of the article.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fd5e10cef94aabc96a0eebe9fd63e7145968e882/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b639dddb-eade-4659-bf8e-ac89b7ffd11d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by jacobdevera — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a1aca081dede1cdd44b4e55b80ea042994d79a9/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a5bd8e1-7c4d-4067-80d2-35ee8472f3ff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aface90a8959533c7e20d609392242d6a6cf9d42/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 85 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2bd4cb56-6e22-4b49-a880-3e70ca9e9476) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**